packages/plugin-protocol-http/src/form.spec.ts

```diff
@@ -0,0 +1,40 @@
+import createForm from './form';
+
+describe('plugins/http/form', () => {
+    it('should transform payload into multi-part form-encoded view', () => {
+        const form = createForm({ a: 1, b: [1, 2, 3], c: 'test' });
+        const boundary = form.getBoundary();
+
+        expect(form.getHeaders()).toEqual({
+            'content-type': expect.stringContaining('multipart/form-data'),
+        });
+
+        expect(
+            form
+                .getBuffer()
+                .toString()
+                .replace(/\r\n/g, '\n')
+        ).toEqual(`--${boundary}
```

Comment:
In this case simple used .toMatchInlineSnapshot()

Reply:
no, form.getBoundary() returns random data for new forms,
that's mean without mocking `form-data` lib there is no way of use snapshots